ouroboros-consensus/docs/Onboarding.md

```diff
@@ -0,0 +1,144 @@
+# Onboarding to the Consensus Team
+
+The content in this file should help orient a new contributor to the Consensus
+Layer: it's fundamental purpose and its neighboring components.
+
+Please open PRs to add particularly useful content here. Though we would like
+the file to remain as approachable as possible, therefore small and
+narrowly-scoped.
+
+## Very High-Level Motivation
+
+At a very high level, a net of Cardano nodes should exhibit the following
+concrete behaviors.
+
+  * The net should diffuse transactions throughout the net as rapidly as
+    possible.
+
+  * The net should diffuse blocks through the net as rapidly as possible.
+
+  * Whenever the ledger rules specify that (a stake pool operating) a particular
+    node should lead, that node should extend the best chain it has seen so far
+    by minting a new block, which should contain as many transactions as
+    possible.
+
+  * The net should be very difficult to disrupt. For example, a successful
+    denial of service attack should be prohibitively expensive to enact.
+
+The primary data are _blocks_ and _transactions_. The possible contents of both
+of those are primarily constrained by the ledger rules. The consensus protocol
+determines how to diffuse those data and how to determine which received chain
```

Comment:
I don't know what it means to say that "the consensus protocol determines how to diffuse this data". The network layer decides how to diffuse data, the consensus protocol doesn't say anything about that.